pkg/controllers/common/constants.go

```diff
@@ -116,6 +116,10 @@ const (
 	TemplateGeneratorMergePatchAnnotation = FederateControllerPrefix + "template-generator-merge-patch"
 
 	LatestReplicasetDigestsAnnotation = DefaultPrefix + "latest-replicaset-digests"
+
+	// NoSyncAnnotation indicates skip syncing. This annotation should only be added manually and temporarily in particular cases.
+	// It works only when workloads have not been propagated to member clusters.
+	NoSyncAnnotation = DefaultPrefix + "no-sync"
```

Comment:
Seems like the annotation should be in Internal group 
![image](https://github.com/kubewharf/kubeadmiral/assets/74236054/c35a9bb1-f881-4aff-a4ef-8fc070e2daf5)